ambry-router/src/main/java/com.github.ambry.router/GetBlobInfoOperation.java

```diff
@@ -43,7 +43,7 @@
  * and completing it. A GetBlobInfo operation only needs to make requests for a single chunk to get the BlobInfo -
  * which is either the only chunk in the case of a simple blob, or the metadata chunk in the case of composite blobs.
  */
-class GetBlobInfoOperation extends GetOperation<BlobInfo> {
+class GetBlobInfoOperation extends GetOperation<GetBlobResult> {
```

Comment:
If both subclasses are working with `GetBlobResult`, then `GetOperation` does not really have to be templatized anymore, does it?